src/common/components/vanilla/button/button.css

```diff
@@ -44,53 +38,38 @@ $neutral-border-color: $dark-grey;
   text-decoration: none;
 }
 
-.primary {
-  composes: button;
-
-  color: $primary-button-text-color;
-  background-color: $primary-button-color;
-}
-
-.primary:active,
-.primary:focus,
-.primary:hover {
-  background-color: $primary-button-color-hover;
-}
-
-.secondary {
+.base {
   composes: button;
-
-  color: $secondary-button-text-color;
-  background-color: $secondary-button-color;
-  border: 1px solid $secondary-border-color;
 }
 
-.secondary:active,
-.secondary:focus,
-.secondary:hover {
-  background-color: $secondary-button-color-hover;
+.base:active,
+.base:focus,
+.base:hover {
```

Comment:
I should dust out the "LoVe HAte" mnemonic ;)
I think it matters when there are different styles applied (in separate selectors).